docs/_docs/developers-guide/clients/cpp.adoc

```diff
@@ -147,24 +147,27 @@ Here is how the client connector configuration looks like:
 
 [source, json]
 ----
-"clientConnector": {
-  "port": 10800,
-  "idleTimeout":3000,
-  "sendServerExceptionStackTraceToClient":true,
-  "ssl": {
-    enabled: true,
-    clientAuth: "require",
-    keyStore: {
-      path: "KEYSTORE_PATH",
-      password: "SSL_STORE_PASS"
-    },
-    trustStore: {
-      path: "TRUSTSTORE_PATH",
-      password: "SSL_STORE_PASS"
+{
```

Comment:
Since we're fixing it anyway, we could remove quotations from the example, making it a proper HOCON rather than a JSON. Maybe it's worth doing as a separate JIRA, to make merge conflicts simpler. What do you think?